### PR TITLE
Use fully qualified syntax for some calls.

### PR DESCRIPTION
**Pull request by jimblandy**
Without this change, rust-analyzer cannot determine the type of
cmd_buf.

The rust-analyzer bug is reported as rust-lang/rust-analyzer#12247.

### CODE REVIEW

File: wgpu-core/src/command/compute.rs [hunk 1]
```diff
@@ -309,8 +309,10 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         let (device_guard, mut token) = hub.devices.read(&mut token);
 
         let (mut cmd_buf_guard, mut token) = hub.command_buffers.write(&mut token);
-        let cmd_buf = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id)
-            .map_pass_err(init_scope)?;
+        let result = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id);
+        // Call using fully qualified syntax, to placate rust-analyzer.
+        // https://github.com/rust-lang/rust-analyzer/issues/12247
+        let cmd_buf = MapPassErr::<&mut CommandBuffer<A>, _>::map_pass_err(result, init_scope)?;
```
Comment (cwfitzgerald) [inline wgpu-core/src/command/compute.rs:315]: How about?

I've actually done this for a lot of other things in my tracker refactor, as I was touching all that code

```suggestion
        let cmd_buf: &mut CommandBuffer<A> = CommandBuffer::get_encoder_mut(&mut *cmd_buf_guard, encoder_id)
```

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]

File: wgpu-core/src/command/compute.rs (hunk 1 above)
Comment (jimblandy) [reply wgpu-core/src/command/compute.rs]: That works too.

**Review by cwfitzgerald — APPROVED** [review]
Format mad, but I'm not

**Comment by jimblandy** [comment]
d'oh

**Pull request merged**